src/app/components/layout/navbar/navbar.component.ts

```diff
@@ -153,7 +153,12 @@ export class NavbarComponent implements OnInit, AfterViewInit, OnDestroy {
       }
       this.status.changeMessage(true);
       this.pingData = data;
-      this.uptime = Utils.secondsToDhms(data['uptime']).roundOffTime;
+      const dayCount = Utils.secondsToDhms(data['uptime']).days;
+      let dayLabel = 'days';
+      if (dayCount === 0 || dayCount === 1) {
+        dayLabel = 'day';
+      }
+      this.uptime = dayCount + ' ' + dayLabel + ' ' + Utils.secondsToDhms(data['uptime']).roundOffTime;
```

Comment:
```
let dayLabel = 'day';
if (dayCount > 1) {
  dayLabel = dayLabel + 's';
}
if(dayCount > 0) {
   this.uptime = dayCount + ' ' + dayLabel + ' ' + Utils.secondsToDhms(data['uptime']).roundOffTime;
}
else {
   this.uptime = Utils.secondsToDhms(data['uptime']).roundOffTime;
}
```

BTW for 0 its days not day. but we omit that, And only put day label if above 24 hours.